source/session.ts

```diff
@@ -316,15 +316,15 @@ export class Session {
         // to timezone naive string.
         return {
           __type__: "datetime",
-          value: data.utc().format(ENCODE_DATETIME_FORMAT),
+          value: date,
```

Comment:
There is a slight difference in the format. `ENCODE_DATETIME_FORMAT` lacks any timezone information. How is this handled server-side?

Reply:
just fine! 
<img width="945" alt="image" src="https://user-images.githubusercontent.com/2225828/219980167-f96c613f-b1ab-4add-8d92-c2e61c011fd2.png">